src/context/auth.js

```diff
@@ -17,11 +17,11 @@ const AuthProvider = ({ children }) => {
     useEffect(() => {
         const storedToken = localStorage.getItem('token')
 
-        if (storedToken) {
+        if (storedToken && !token) {
```

Comment:
What was the context of this change? Why was this needed?

Reply:
We also did not touch this file, maybe we did a git pull from the main branch and pulled someone elses work by mistake